notification/src/vm2SandboxExecutor.ts

```diff
@@ -0,0 +1,38 @@
+import { VM } from 'vm2'
+
+import SandboxExecutor from './interfaces/sandboxExecutor';
+
+
+const FUNCTION_WRAP_PREFIX_LENGTH = 1
+
+export default class VM2SandboxExecutor implements SandboxExecutor {
+  vm: VM;
+
+  constructor (timeout = 5000) {
+    this.vm = new VM({
+      timeout
+    })
+  }
+
+  evaluate (expression: string, data: object): boolean {
+    const wrapper =
+      'f=function(data){' +
+      'return ' +
+      expression +
+      '};f(' +
+      JSON.stringify(data) +
+      ');'
+
+    let result = false
+    try {
+      result = this.vm.run(wrapper)
+    } catch (err) {
+      // console.error('Malformed expression received: ' + expression, err)
```

Comment:
uncommenting this and logging the error will probably be helpful

Reply:
Issue was that there is no logger installed I think. See conversation below! (throwing error now)

Reply:
perfect